polaris.shopify.com/.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "search.exclude": {
+    "**/.next": true,
+    "**/node_modules": true
+  }
+}
```

Comment:
I think `.gitignore` should remove files from vscode search. Is something else going on?